src/v/redpanda/admin_server.cc

```diff
@@ -1836,6 +1836,13 @@ void admin_server::register_features_routes() {
                   throw ss::httpd::bad_request_exception(
                     fmt::format("License is expired: {}", license));
               }
+              const auto& ft = _controller->get_feature_table().local();
+              const auto& loaded_license = ft.get_license();
+              if (loaded_license && (*loaded_license == license)) {
+                  /// Loaded license is idential to license in request, do
+                  /// nothing and return 200(OK)
+                  co_return ss::json::json_void();
```

Comment:
I was thinking about uploading HTTP 201 to indicate that the requested license is already set, however seems a bit complicated, i'll need to use the `register_routes_raw` method and manually set the reply data and code. 

Anyone opposed to just returning 200? The small negative is the client will just observe a successful PUT even though technically nothing was done.